src/BloomBrowserUI/pageChooser/js/page-chooser.js

```diff
@@ -43,6 +44,40 @@ var PageChooser = (function () {
         $("#preview").attr("src", $(this._selectedGridItem).find("img").first().attr("src"));
         this.setLocalizedText($('#previewDescriptionText'), 'TemplateBooks.PageDescription.', $(".pageDescription", this._selectedGridItem).text(), defaultCaptionText);
     }; // thumbnailClickHandler
+    // There's a bug deep in javascript that doesn't take into account the scrolling
+    // of a div element before something inside it is clicked on.  The following code
+    // detects whether the scrolling has changed since the last mouse click, and if so,
+    // searches for the item which should have matched.  For the initial bug report,
+    // see https://silbloom.myjetbrains.com/youtrack/issue/BL-2623.
+    // Note that the offset().top values returned by jquery properly take into account
+    // the scrollTop of the scrolling parent div.  Which makes me think the bug may be
+    // below the jquery level!?
+    PageChooser.prototype.findProperElement = function (clickedDiv, evt) {
+        var gridItem = $(clickedDiv).parent();
```

Comment:
Is it definitely not possible to click somewhere not in any grid item? e.g. on the right of the bottom row if it is not full? Do we really want to return the parent div (and add class ui-selected to it) if that happens?